localstack/services/secretsmanager/provider.py

```diff
@@ -314,23 +314,6 @@ def fake_secret_update(
         self.last_changed_date = time.time()
 
 
-class FakeSecretVersionStore(dict):
```

Comment:
I've ran some secretsmanager tests and it seems they run without the patch now, I think it has been fixed upstream with https://github.com/getmoto/moto/pull/6720